src/moa/moa_controllers/moa_controllers/ack_to_motec.py

```diff
@@ -34,25 +34,28 @@ def __init__(self):
         )
 
         #create server for CAN
-        self.server = self.create_service( #Unsure if needs to be server or client
+        self.can_client = self.create_client( #Unsure if needs to be server or client
             CANSendReq,
-            'CAN_SEND_SRV',
-            self.can_srv_callback
+            'CAN_SEND_SRV'  
         )
 
+        self.req = CANSendReq.Request()
```

Comment:
This node needs to wait for it to get the service connection else it will crash. example implimentation is:
```
while not self.cli.wait_for_service(timeout_sec=1.0):
            self.get_logger().info('service not available, waiting again...')
```